src/fabric/src/fabric_doc_open.erl

```diff
@@ -517,6 +526,24 @@ t_read_repair() ->
         ?assertEqual(bar, read_repair(Acc4))
     end).
 
+t_no_read_repair_for_replicator() ->
+    Foo1 = {ok, #doc{revs = {1, [<<"foo">>]}}},
+
+    ?_test(begin
+        meck:expect(couch_log, notice, fun(_, _) -> ok end),
+        meck:expect(couch_stats, increment_counter, fun(_) -> ok end),
+
+        % Test no read_repair too place
+        meck:reset(fabric),
+        Acc0 = #acc{
+            dbname = <<"dbname1/_replicator">>,
+            replies = [fabric_util:kv(Foo1, 1)]
+        },
+        ?assertEqual(Foo1, read_repair(Acc0)),
+        timer:sleep(100),
```

Comment:
This is a little unfortunate. I suppose an alternative might be to log something at debug level for a replicator db, and wait until that log line gets triggered?

Reply:
I think it's not worth adding a log line just for the test, it should be genuine need for it. In case where we want to show that an async function "wasn't" called, is probably one of the few case when I can still see a sleep being the only option.